packages/airnode-deployer/terraform/airnode/aws/templates/httpGw.yaml.tpl

```diff
@@ -96,3 +103,34 @@ paths:
         responses:
           default:
             statusCode: 200
+    options:
+      parameters:
+        - $ref: "#/components/parameters/endpointId"
+      responses:
+        "200":
+          description: CORS preflight response
+          headers:
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/EndpointResponse"
+      x-amazon-apigateway-request-validator: none
```

Comment:
This is also to ensure that preflight checks are only passed through the the lambda without any validation of  the shape

Reply:
Why is this needed?

Reply:
The preflight check doesn't pass the request body and parameters validation so this disabled it

Reply:
So it turns out that this is infact not needed (just tested a deployment), I've removed it.